Libplanet/Net/Swarm.cs

```diff
@@ -57,14 +57,36 @@ static Swarm()
             }
         }
 
+        /// <summary>
+        /// Creates a <see cref="Swarm{T}"/>.  This constructor in only itself does not start
+        /// any communication with the network.
+        /// </summary>
+        /// <param name="blockChain">A blockchain to publicize on the network.</param>
+        /// <param name="privateKey">A private key to sign messages.  The public part of
+        /// this key become a part of its end address for being pointed by peers.</param>
+        /// <param name="appProtocolVersion">An app protocol to comply.</param>
+        /// <param name="workers">The number of background workers (i.e., threads).</param>
+        /// <param name="host">A hostname to listen.</param>
```

Comment:
Actually, we aren't using `host` as an argument for traditional server socket but use for indicating its public endpoint that can be connected by other peers. 😕